src/components/Board.js

```diff
@@ -5,28 +5,130 @@ import axios from 'axios';
 import './Board.css';
 import Card from './Card';
 import NewCardForm from './NewCardForm';
-import CARD_DATA from '../data/card-data.json';
+
+
+
+
+const CARDS_URL = 'https://inspiration-board.herokuapp.com/boards/brenda/cards'
+
 
 class Board extends Component {
+
+
   constructor() {
     super();
 
     this.state = {
       cards: [],
+      status: {
+        message: "Successfuly cards loaded!",
+        type: "success"
```

Comment:
I think we could remove this `status` part of the `Board` component's state. This is because we're already tracking the status message state in the `App` component (and indeed, that's where the data is being used to render the `Status` component).

So basically, this bit of state data is redundant and is not currently being used by your code.